### DISCUSSION TITLE
Getting Started with Discussions

### DISCUSSION THREAD

**Discussion by Nelson-Gon** [Nelson-Gon/urlfix#6] [General]
Hello there open source contributor,

Thank you for your interest in `urlfix`.  Your contribution is highly valued and we would love to hear from you. 

Please feel free to use this space to:

  * Ask questions that may not be suitable to open as [issues](https://github.com/Nelson-Gon/urlfix/issues) 
  * Share ideas that you would like to first discuss before opening as an enhancement [issue](https://github.com/Nelson-Gon/urlfix/issues?label=enhancement)
  * Engage with other community members.

For discussions that involve a specific issue or pull request, please link to that issue/pull request in your thread.

Thank you,

Team urlfix

**Comment by akhilkumarjilakara** [comment]
Hi, 

I have cloned your code and running the install command "python3 setup.py install"

I am doing "from urlfix import *"

and while calling the "URLFix"  i am getting the bellow issue.

from the below src you can get some image.

<img width="475" alt="urlfix_error" src="https://user-images.githubusercontent.com/25062937/109840527-3907ee80-7c6e-11eb-8939-45531d735823.PNG">


the error in text : (i have run this in python3 CLI) and the system is a VM.

Python 3.8.5 (default, Jan 27 2021, 15:41:15) 
[GCC 9.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from urlfix import *
>>> fix = URLFix("testurls.txt", output_file="replacement.txt")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
NameError: name 'URLFix' is not defined
>>>

**Reply by Nelson-Gon** [reply]
Hi,

Thanks for the feedback. I am currently unable to test in a VM (1am here) but does calling `urlfix.URLFix` work? What of `from urlfix import urlfix` and then `urlfix.URLFix`?! I think you should open an (bug) issue and title it as "Issues with import" (or a title you like).
Not sure why it doesn't work since it works in Actions.

**Reply by akhilkumarjilakara** [reply]
from urlfix import urlfix and then urlfix.URLFix

with the above line it's working fine.

you, still want me to open a issue ?

**Reply by Nelson-Gon** [reply]
Awesome, I think I'll update docs to use explicit imports instead of (bad) * imports. Glad it worked and happy to answer further questions. I think there's no need to open an issue now if it worked, I'll update docs and create a wiki with FAQs and include this.
